draft-ietf-ppm-dap.md

```diff
@@ -358,10 +358,112 @@ client could not report 10^6s or -20s.
 
 # Message Transport
 
-Communications between DAP entities are carried over HTTPS {{!RFC2818}}. HTTPS
-provides server authentication and confidentiality. When client authenticaiton
-is also required, the client uses the mechanism described in
-{{https-sender-auth}}.
+Communications between DAP participants are carried over HTTPS {{!RFC2818}}.
+HTTPS provides server authentication and confidentiality. Use of HTTPS is
+REQUIRED.
```

Comment:
I think there's good arguments against relying on a transport-level concept of identity to authenticate protocol-level concepts, but [this is good enough for ACME](https://www.rfc-editor.org/rfc/rfc8555.html#section-6) so perhaps we can lean on that precedent.

Reply:
Ultimately this question would be answered by a proof of (in)security in a formal model. I'm perfectly happy to make this a requirement for now.

One nit: Does RFC2818 cover HTTP/3? I'm wondering if @martinthomson can help us spell this requirement :)

Reply:
You can start to refer to RFC 9110 for that stuff now.  That definitely covers HTTP/3.

(Technically, RFC2818 did cover HTTP/3, but only circuitously.  Also, RFC 2818 is a little old in a few important ways.)